OOVPADatabase/D3D8/3911.inl

```diff
@@ -417,6 +417,35 @@ OOVPA_NO_XREF(D3DDevice_GetGammaRamp, 3911, 14)
         { 0x27, 0xC2 },
 OOVPA_END;
 
+// ******************************************************************
+// * D3D_DestroyResource
+// ******************************************************************
+// Generic OOVPA as of 3911 and newer.
+OOVPA_NO_XREF(D3D_DestroyResource, 3911, 38)
+    // push edi 
+    // push eda
+    // mov edi [esp + $0C]
+    // mov eax, [edi]
+    // mov esi, eax
+    OV_MATCH(0x00, 0x56, 0x57, 0x8B, 0x7C, 0x24, 0x0C, 0x8B, 0x07, 0x8B, 0xF0),
+
+    // and esi, $70000
+    // cmp esi, $50000
+    OV_MATCH(0x0A, 0x81, 0xE6, 0x00, 0x00, 0x07, 0x00, 0x81, 0xFE, 0x00, 0x00, 0x05, 0x00),
+    
+    // jne eip + $04
+    // test eax, eax
+    // jne eip + $06
+    // push edi
+    OV_MATCH(0x16, 0x75, 0x04, 0x85, 0xC0, 0x79, 0x06, 0x57),
+
+    // cmp esi, $50000
+    OV_MATCH(0x28, 0x81, 0xFE, 0x00, 0x00, 0x05, 0x00),
```

Comment:
Again, is also unique. Though there is no and opcode before this which is a good thing.